bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/link/ItemChannelLinkResource.java

```diff
@@ -174,7 +180,13 @@ public Response link(@PathParam("itemName") @Parameter(description = "itemName")
                     @ApiResponse(responseCode = "405", description = "Link not editable.") })
     public Response unlink(@PathParam("itemName") @Parameter(description = "itemName") String itemName,
             @PathParam("channelUID") @Parameter(description = "channelUID") String channelUid) {
-        String linkId = AbstractLink.getIDFor(itemName, new ChannelUID(channelUid));
+        ChannelUID uid;
+        try {
+            uid = new ChannelUID(channelUid);
+        } catch (IllegalArgumentException e) {
+            return Response.status(Status.BAD_REQUEST).build();
```

Comment:
@kaikreuzer : code 400 is not described as possible return code of the API. It has to be added.